data/companies.json

```diff
@@ -544,6 +544,23 @@
         "youtube": null
       }
     },
+    {
+      "name": "Karobar App",
+      "logoName": "karobar_app.png",
+      "establishedAt": null,
```

Comment:
The establishedAt is a required field and is a unix timestamp. Please update it accordingly!